openfisca_aotearoa/variables/live_organ_donor_compensation.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Person
+from numpy import round
+
+class weekly_compensation_before_tax(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR # TODO - determine whether we need to get WEEK to work
```

Comment:
Is this comment still relevant? It seems this code works well with `YEAR`, even though it would be more proper with `WEEK`.

Reply:
In the current scheme, ETERNITY may be better since the value doesn't correspond to a specific MONTH or YEAR. 

I think the value actually corresponds to a particular point in time corresponding to the date of the latest pay period that has been used in the calculation.